buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/publish/PublishingPlugin.groovy

```diff
@@ -155,6 +178,13 @@ class PublishingPlugin implements Plugin<Project> {
         }
     }
 
+    private static Set<String> changedPaths(File workDir) {
```

Comment:
afaicr, we have the same git command elsewhere, can we centralize this in one place to avoid duplication?